js/constants/config.js

```diff
@@ -7,7 +7,7 @@
 // https://vault-staging.brave.com for a dev build
 // http://localhost:3000 for production
 const vaultHost = process.env.VAULT_HOST || 'https://vault-staging.brave.com'
-const adHost = process.env.AD_HOST || '//cdn.brave.com'
+const adHost = process.env.AD_HOST || '//oip.brave.com'
```

Comment:
Well, the point is that the browser should consider anything it exposes to be "public knowledge" and make decisions accordingly. I thought there was an additional cost in dollars at Fastly for SSL and that's what I was concerned about, but looking back at their pricing page I guess there isn't so I suppose there is no downside to always having SSL always on